### PR TITLE
Add proper README documentation

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,5 +54,110 @@ spec:
...
 ```
+
+Additionally, container image used to deliver this plugin can be set using
+`LINUX_BRIDGE_IMAGE` environment variable in operator deployment manifest.
+
+# Deployment
+
+First install the operator itself:
+
+```shell
```
Comment (davidvossel) [inline README.md]: just a quick thought, it would be nice if all of this could be condensed into a single manifest.

Comment (phoracek) [reply README.md]: Sure, sounds good (and I believe it won't break OLM integration)

Comment (phoracek) [reply README.md]: Actually, I think we have to explicitly create the namespace before deployment using it, right @davidvossel? At least in older versions of Kubernetes I had that problem.

Comment (davidvossel) [reply README.md]: yep, the namespace has to exist before we can post a Deployment in the namespace. 

take a look at the operator manifest kubevirt has. The namespace is created for the operator to be deployed into.

https://github.com/kubevirt/kubevirt/blob/master/manifests/release/kubevirt-operator.yaml.in

when you all get around to looking at olm integration and writing the CSV file, I believe it is expected that the namespace already exists. That would be something to look into once you get to that point.

**Pull request merged**